src/java.base/share/classes/java/lang/reflect/MalformedParameterizedTypeException.java

```diff
@@ -50,6 +50,8 @@ public MalformedParameterizedTypeException() {
      * Constructs a {@code MalformedParameterizedTypeException} with
      * the given detail message.
      * @param message the detail message; may be {@code null}
+     *
+     * @since 10
      */
     public MalformedParameterizedTypeException(String message) {
```

Comment:
Both this and the explicit no-arg constructor were introduced in this class through https://bugs.openjdk.org/browse/JDK-8183175 in Java 10. Since an (implicit) no-arg constructor was always available even before that change, it makes sense to add a `@since 10` to only this explicit constructor. This change looks good.